arch/M68K/M68KDisassembler.h

```diff
@@ -18,6 +18,8 @@ typedef struct m68k_info {
 	unsigned int type;
 	unsigned int address_mask; /* Address mask to simulate address lines */
 	cs_m68k extension;
+	uint8_t groups[8];
+	uint8_t groups_count;
```

Comment:
oh yes it seems so. then would you mind extending `test_m68.py` to handle these instruction groups, so it does exactly what `test_m68.c` does?